warning: '_WIN64' is not defined, evaluates to 0 [-Wundef]

On FreeBSD I'm getting these warnings:
```
In file included from /usr/ports/science/py-qiskit-aer/work/qiskit-aer-0.2.1/src/simulators/qasm/qasm_controller.hpp:18:
/usr/ports/science/py-qiskit-aer/work/qiskit-aer-0.2.1/src/base/controller.hpp:29:7: warning: '_WIN64' is not defined, evaluates to 0 [-Wundef]
#elif _WIN64
      ^
/usr/ports/science/py-qiskit-aer/work/qiskit-aer-0.2.1/src/base/controller.hpp:376:7: warning: '_WIN64' is not defined, evaluates to 0 [-Wundef]
#elif _WIN64
      ^
2 warnings generated.
```

Are you by any chance developing the project on Windows?

--- a/src/base/controller.hpp
+++ b/src/base/controller.hpp
@@ -26,7 +26,7 @@
 
 #if defined(__linux__) || defined(__APPLE__)
    #include <unistd.h>
-#elif _WIN64
+#elif defined(_WIN64)
    // This is needed because windows.h redefine min()/max() so interferes with std::min/max
    #define NOMINMAX
    #include <windows.h>
@@ -373,7 +373,7 @@ size_t Controller::get_system_memory_mb(void){
    auto pages = sysconf(_SC_PHYS_PAGES);
    auto page_size = sysconf(_SC_PAGE_SIZE);
    total_physical_memory = pages * page_size;
-#elif _WIN64
+#elif defined(_WIN64)
    MEMORYSTATUSEX status;
    status.dwLength = sizeof(status);
    GlobalMemoryStatusEx(&status);

